include/NovelRT/Maths/GeoVector2F.h

```diff
@@ -4,9 +4,23 @@
 #ifndef NOVELRT_MATHS_GEOVECTOR2_H
 #define NOVELRT_MATHS_GEOVECTOR2_H
 
-#ifndef NOVELRT_H
-#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
-#endif
+#include <glm/glm.hpp>
+#include <glm/gtx/rotate_vector.hpp>
+
+namespace NovelRT::Graphics
+{
+    class RenderObject;
+    class ImageRect;
+    class BasicFillRect;
+    class TextRect;
+    class RenderingService;
+    class Camera;
+} // namespace NovelRT::Graphics
+
+namespace NovelRT::Input
+{
+    class InteractionService;
+}
```

Comment:
Why is this in `GeoVector2F`?

Reply:
This is a leftover from when we were unsure how we wished to structure the namespace headers, and was from the brief period I made headers self-contained.